subsys/net/ip/tcp2.c

```diff
@@ -413,6 +417,11 @@ static void tcp_send_process(struct k_work *work)
 	if (conn && conn->in_retransmission) {
 		k_delayed_work_submit(&conn->send_timer, K_MSEC(tcp_rto));
 	}
+
+out:
```

Comment:
Maybe `unlock` will be a more meaningful name?

Reply:
We have used the out: label in other functions so I would call this same here.